daemon/logs.go

```diff
@@ -101,9 +104,11 @@ func (daemon *Daemon) ContainerLogs(name string, config *ContainerLogsConfig) er
 				}
 				cLog = tmp
 			}
+
 			dec := json.NewDecoder(cLog)
 			l := &jsonlog.JSONLog{}
 			for {
+				l.Reset()
```

Comment:
Yea, but in looking at the code between here and there, nothing is set/changed in 'l' so I'm wondering why it needed to be moved.  I kind of wonder if this is fixing a potential bug that's been there for a while.